vault/resource_jwt_auth_backend.go

```diff
@@ -280,6 +281,29 @@ func jwtAuthBackendRead(d *schema.ResourceData, meta interface{}) error {
 
 }
 
+func convertProviderConfigValues(input map[string]interface{}) (map[string]interface{}, error) {
+	newConfig := make(map[string]interface{})
+	for k, v := range input {
+		switch k {
```

Comment:
Nit: you can set (and check) against `v` outside of the switch to avoid type checking inside it via `val, ok := v.(string)`.

Reply:
Done!